src/components/BotBuilder/BotBuilder.css

```diff
@@ -299,6 +299,14 @@ input[type=file]{
 	}
 }
 
+@media  (max-width:769px){
+	.botbuilder-col{
+		position: inherit !important ;
```

Comment:
Is important necessary?

Reply:
Yes `!important` is necessary without that the value of position is `fixed`.